providers/src/airflow/providers/edge/cli/edge_command.py

```diff
@@ -191,6 +201,64 @@ def _get_sysinfo(self) -> dict:
             "free_concurrency": self.free_concurrency,
         }
 
+    def _launch_job(self, edge_job: EdgeJobFetched):
+        """Get the received job executed."""
+        process: Popen | Process
+        if AIRFLOW_V_3_0_PLUS:
```

Comment:
Nit: Could we pull each of these into separate functions -- `def _launch_job_airflow_v3` etc?

Reply:
Adjusted. Good idea!